docs/spec_draft.md

```diff
@@ -254,6 +311,54 @@ the (real, imaginary) pairs.
 // %result: [[1, 2], [3, 4]]
 ```
 
+## stablehlo.negate
+
+`stablehlo.negate(operand) -> result`
+
+### Semantics
+
+Performs element-wise negation of `operand` tensor and produces a `result`
+tensor. For complex types, the operation negates both the real and the imaginary
+parts.
```

Comment:
This also seems to be part of calculus textbooks. E.g. the first result in my Google search says: https://www2.clarku.edu/~djoyce/complex/plane.html#:~:text=Negation%20and%20Subtraction.&text=Of%20course%2C%20the%20negation%20of,left%20and%20one%20unit%20down.